src/server.c

```diff
@@ -220,10 +252,8 @@ static inline void handle(Server *server, int fd, fd_set *activeFDs, struct sock
             requestDel(req);
         }
     }
-
+    serverDelFd(server->epollfd, fd);
```

Comment:
maybe you've better to pass server instead of epollfd because you need to add `#ifndef _WIN32`